stable/aws-load-balancer-controller/values.yaml

```diff
@@ -87,3 +87,9 @@ livenessProbe:
     scheme: HTTP
   initialDelaySeconds: 30
   timeoutSeconds: 10
+
+extraArgs: {}
+
+extraEnv: {}
```

Comment:
add comment similar to - 
Environment variables to set for aws-load-balancer-controller pod

Reply:
Fixed :+1: